### ISSUE TITLE
Responsive UI

### ISSUE THREAD

**Issue by cunidev** [glimpse-editor/Glimpse#100]
Clearly not a priority for 0.1, but designing Glimpse with responsive UI (e.g. libhandy widgets) should be definitely considered, in order to at least adapt the UI (mockups) to mobile devices easily in the future and exploit Gtk's full touch input potential.

**Comment by Member1221** [comment]
I don't see how useful an application like this would be on a mobile device. I guess if the screen has like, sensitivity it could be useful for artists.

I think such a tool would warrant its own small app at some point, could be something to discuss in the 2.x stream once we get there.

**Comment by cunidev** [comment]
It would indeed, as popular apps like Snapseed, Photoshop Express etc. have proved.
Actually it does not have to be a primarily mobile-oriented UI. Once menus and dialogs are easily accessible by touch input devices, and the UI allows for a portrait-ish mode, we have a world of new devices and capabilities without any extreme effort.

**Comment by Member1221** [comment]
Yeah. Though we should make sure that we don't sacrifice desktop usability for phone usability, hence why it's best that at that point we'd be making a phone specific side app optimized for a phone's navigation style.
Especially since phone screen sizes vary and we'd have to make sure that the app doesn't look really weird on a phone, while also (if it was to be intergrated straight in to the main app) we'd have to make sure that the phone navigation wouldn't be a fustration to the user.

**Comment by TrechNex** [comment]
For now this is still a UI rewrite candidate, but we could bring it in as a Fork Candidate if upstream eventually release v3.0.

libhandy depends on GTK3, but they currently use GTK2 for the 2.10.x base.

**Comment by jannuary** [comment]
Snapseed and PSx are both very different from any desktop image editor and I'm not sure it's worth pursuing given the very limited appeal a potential Glimpse for phones would have. I think it's much better to can be used on desktop as well as, say, touch-enabled tablets and devices with small screen sizes, which GTK and ~libhandy~~ libadwaita could help quite a bit with. 

Relevant mockup [here.](https://github.com/glimpse-editor/glimpse-nx-design/blob/main/primary-window/adaptive-experiments-jannuary.png)
